spec/controllers/catalog_controller_spec.rb

```diff
@@ -137,6 +137,14 @@
         put :search, query: 'query'
         expect(assigns(:equipment_model_results)).to eq([@equipment_model])
       end
+      it 'should give unique results even with multiple matches' do
+        @equipment_model = FactoryGirl.create(:equipment_model,
+                                              active: true,
+                                              name: 'query',
+                                              description: 'query')
+        put :search, query: 'query'
+        expect(assigns(:equipment_model_results)).to eq([@equipment_model])
```

Comment:
I think we might want to test that `assigns(:equipment_model_results).uniq` is equal to `assigns(:equipment_model_results)` as well (to verify that there are no duplicates). I know it's a bit redundant but it more directly tests for the original bug; what do you think?